i18n/en.toml

```diff
@@ -218,3 +218,6 @@ other = "Warning:"
 
 [whatsnext_heading]
 other = "What's next"
+
+[3rdparty_message]
+other = """This section links to third party projects that provide functionality required by Kubernetes. The Kubernetes project authors aren't responsible for these projects. This page follows <a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">CNCF website guidlines</a> by listing projects alphabetically. To add a project to this list, read the <a href="/contribute/style/content-guide/#third-party-content">content guide</a> before submitting a change."""
```

Comment:
I'd suggest we revise "functionality required by Kubernetes" to something more inclusive so that we can use the shortcode in other contexts. There are projects not strictly required by Kubernetes, including the service catalog, the client libraries, etc. However, they are very useful when developing and deploying workloads into a Kubernetes cluster.

Reply:
Typo: guidlines.

Reply:
@tengqm we could extend this shortcode with parameters to declare required vs. optional functionality - if we do, I'd defer that to a follow-up PR.

Reply:
Let's get this text right before the content gets localized.
Can this string be broken up into several lines or a few variables? It is difficult to read.
Instead of variables, you could create a snippet of Markdown (that gets localized) and
included in the shortcode?
Also, the URLs do not require localization. The long string could be concatenated from several variables.
You may want to reduce the text if you plan to define the content in a single variable.

Some edits:
This section links to third party content that is not maintained by the Kubernetes project.

[Since, you provide the link to the docs content guide, could this next sentence appear in the content guide instead of in the shortcode?]
This page follows the <a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">CNCF website guidelines</a> by listing projects alphabetically.

[Create a sentence that can be easily constructed]
Before submitting a change to add third party content, read the <a href="/contribute/style/content-guide/#third-party-content">Kubernetes docs content guide</a>.

Reply:
@kbhawkey Due to the way the i18n variables work, no – they want a string so return characters tend to mess things up a bit.

In regards to URLs in the translation strings – a number of other translation strings have URLs in them? I'm hesitant to break with tradition here, moreover I think introducing multiple translation strings just for the purpose of excluding URLs, is, at our current stage of translation maturity, extra work.

As for wording: I brought this up during the last SIG Docs meeting (prior to Kubecon) in the issue. I'm happy to bring it up again this week if we still need to workshop it, but I'd prefer if we kept discussion to the issue rather than in this PR.

Reply:
I added several comments into the issue based upon viewing the shortcode in use.